Orange/misc/bottlechest.py

```diff
@@ -0,0 +1,183 @@
+"""
+This module provides alternatives for the few additional functions found in
+and once used from the bottleneck package (fork of bottleneck).
+
+It also patches bottleneck to contain these functions.
+
+It also makes bottleneck available as bottlechest.
```

Comment:
I don't think anybody outside the core group and core code used bottlechest, hence it may be better to break the compatibility than cause confusion for indefinite future.